resources/tracing/values.yaml

```diff
@@ -235,4 +235,8 @@ authorizationPolicy:
 #      - cluster.local/ns/kiali/sa/kiali-server-service-account
   monitoring:
      extraPrincipals:
-#      - cluster.local/ns/kiali/sa/kiali-server-service-account
\ No newline at end of file
+#      - cluster.local/ns/kiali/sa/kiali-server-service-account
+
+istio:
+  samplingRate: 100.0
```

Comment:
I would turn it around to have the same behaviour as before. The istio chart used 0.1 by default and only changed it for evaluation to 100.0

Reply:
Done